### DISCUSSION TITLE
Large image with scroll vertical in swiper horizontal

### DISCUSSION THREAD

**Discussion by tarcisioandrade** [nolimits4web/swiper#6851] [Q&A]
Hello, I am trying to create a manga reader and I came across a problem using swiperJS Core (no frameworks).

Some images are quite large, Ex: 800x8000, and the default behavior of swipe is to make it "fit" whole on the page, I would like the user to scrollase until finished reading and move to the next page (slide).

# EXPECTED:

![WhatsApp Image 2023-07-12 at 10 16 52](https://github.com/nolimits4web/swiper/assets/61153830/861b0c26-def9-4321-a6bc-71227af041a5)

# REALITY
![WhatsApp Image 2023-07-12 at 10 16 52 (1)](https://github.com/nolimits4web/swiper/assets/61153830/9cd21fee-f13e-4bff-b221-9ee646a05bd1)

Does anyone have any suggestions to get the result I want?

**Comment by ckanitz** [comment]
I'm unsure if I got you correctly but setting a maxHeight and overflow-y: scroll might help?

**Reply by tarcisioandrade** [reply]
I already tried it, it continues with this default behavior of "contain"

**Reply by ckanitz** [reply]
Maybe create a simple codesandbox (https://codesandbox.io) of your approach and I / others can play around with it? (or any similar service)

**Reply by tarcisioandrade** [reply]
Okay, I confess that I'm not using the swiper in a normal environment, I'm rendering in a webview in react-native, but I'd be happy if you could help me.

https://snack.expo.dev/@woltztyger/spunky-bananas2

**Reply by ckanitz** [reply]
The provided code won't render in my browser. Try to break it down to a simple "getting-started-tutorial"-level of a single .html page.
